### PR TITLE
Improve features

### PR DESCRIPTION
**Pull request by TobiGr**
With this PR

- a new display width is added. It optimizes the design for extra small screens, in our case small phones in portrait mode. A width of `< 544px` is addressed, I call it `xss`. An equal width is used in Bootstrap 4 Alpha.

- the design of the features is now adjusted for these extra small devices, too. From now on we hide descriptions for >544px until the user toggles them.

- ~~the gallery is disabled. Only the first slide is shown until we have detailed descriptions and screenshots (see #22 and #13).~~

- the posibility to hide the first slide for certain widths was removed.

- the gallery controls do not cover texts or images.

@TheAssassin Should we merge the `gallery.css` into `style.css` file? I'm not sure because we won't need the gallery somewhere else. At the moment we have code in both CSS files which addresses the gallery and its container.

### CODE REVIEW

**Comment by TobiGr** [comment]
@TheAssassin I enabled the gallery again and made some smaller improvements. Feel free to review.

**Comment by TheAssassin** [comment]
Oh, btw, @theScrabi has redirected me to https://github.com/TeamNewPipe/NewPipe/tree/master/screenshots and said we can just use these, they're probably even updated from time to time. You probably want to use these for the gallery.

> Descriptions:
> 
> 1: Search for videos and channels
>
> 2: Everything you need to see about a video
>
> 3: Even with another skin
>
> 4: Listen and control a video's audio from everywhere.
>
> 5: Watch videos whatever you do, with a popup player.
>
> 6: Browse channels
>
> 7: Download videos and music
>
> 8: Handle downloads with a simple to use download manager.
>
> 9: (you may not need to use this) Search for channels

**Comment by TobiGr** [comment]
It's great that we have new screenshots. But our main blocker are the detailed descriptions for the features.

**Comment by TheAssassin** [comment]
@theScrabi since you provided the screenshots, please post a few more detailed descriptions.

**Comment by theScrabi** [comment]
I send them to you via mail, but I can post them here tomorrow.

**Comment by TheAssassin** [comment]
@theScrabi if you'd had looked more thoroughly, I posted your suggestions above already. @TobiGr needs more detailed descriptions though.

**Comment by theScrabi** [comment]
Idon't want ro be unpolide, but I can't think about a more detailed description at the moment. (In general i don't have much time timr left for NewPipe at the moment since i have my tests for this semester soon). So @TobiGr would you like to find detailed (and marketing like) descriptions to the screenshots?

**Comment by TobiGr** [comment]
@TheAssassin I've worked out some descriptions and more slides. I didn't find a good picture for the 'extended privacy' slide. Do you have any idea what I can use? @theScrabi It would be great if you go through the descriptions to see if everything is correct. If you do not find any false statements or typos I don't have any future commits to do.
Btw. Good luck with your tests!

**Comment by TheAssassin** [comment]
Something like [this](https://upload.wikimedia.org/wikipedia/commons/0/07/Privacy_written_in_tiles.jpg) should work. I don't think you could make a screenshot of privacy.

**Comment by TobiGr** [comment]
Yes. That looks good. I added it and fixed some small bugs.

**Comment by TobiGr** [comment]
@theScrabi screenshot_5.png (the one with the popup) is a little overloaded. If you see it the first time you need to search for the important element. That's not perfect, but is okay a the moment.

**Comment by TheAssassin** [comment]
@TobiGr You should **always** check the license of content, even if someone suggests you to do so. I just posted the link to this specific picture as an example. The file itself is of course copyrighted, and you unfortunately violated the CC-By 2.0 license by not crediting the author. https://commons.wikimedia.org/wiki/File:Privacy_written_in_tiles.jpg

Though, the license allows you to modify the picture. The page contains a recommendation how to credit properly, so I'd suggest you to follow that suggestion, but edit it right into the picture. That is the safest way to ensure compliance.

I feel myself bad for suggesting the picture without giving a hint on how to credit the author properly.

**Comment by TobiGr** [comment]
Sorry for responding that late, but I had some more important things to do.

> I feel myself bad for suggesting the picture without giving a hint on how to credit the author properly.

Don't worry. That is not a problem. `By Owen More via Wikimedia Commons. License: CC BY 2.0` should do it, right?

**Comment by TheAssassin** [comment]
Yes, technically. But as written above, I would just follow the author's suggestion instead. Also, you could omit the Commons part if you want to since you're not required to include it by the author. Please see https://commons.wikimedia.org/wiki/Commons:Credit_line for more information.

File: index.html [hunk 1]
```diff
@@ -262,10 +262,7 @@ <h1 class="hidden-xs">NewPipe</h1>
                                 <div class="container">
                                     <div class="details-flex">
                                         <div class="details-image">
-                                            <figure>
-                                                <img src="img/Privacy_written_in_tiles.jpg"/>
-                                                <figcaption>Photo credit: <a href="www.homewatersoftenerreviews.com">Home Water Softener Reviews</a>. License: <a href="https://creativecommons.org/licenses/by/2.0/">CC BY 2.0</a></figcaption>
-                                            </figure>
+                                            <img src="img/Privacy_written_in_tiles.jpg"/>
```
Comment (TheAssassin) [inline index.html]: Correct me if I'm wrong, but it looks like you removed the credit on GitHub. That doesn't make very much sense.

Comment (TobiGr) [inline index.html]: > Though, the license allows you to modify the picture. The page contains a recommendation how to credit properly, so I'd suggest you to follow that suggestion, but edit it right into the picture. That is the safest way to ensure compliance.

I added `Photo credit: Home Water Softener Reviews. Photo by Owen More. License: CC BY 2.0` into the photo. That's why I removed the HTML code.

Comment (TheAssassin) [inline index.html]: I see, GitHub hid some changes. The above line would've sufficed by the way.

**Comment by TheAssassin** [comment]
Looks like we're ready to merge, aren't we, @TobiGr?

**Comment by TobiGr** [comment]
Yes, it is.

**Pull request merged**